backend/api/job_board/schema.py

```diff
@@ -0,0 +1,49 @@
+from typing import List, Optional
+
+import strawberry
+from job_board.models import JobListing
+
+from .types import JobListing as JobListingType
+
+
+@strawberry.type
+class JobBoardQuery:
+    @strawberry.field
+    def job_listings(self, info) -> List[JobListingType]:
+        return [
+            JobListingType(
```

Comment:
I think we can avoid duplicating this init logic (here and in job_listing detail) with the `from_django_model`: https://github.com/pythonitalia/pycon/pull/2571/files#diff-d65bfc90e5a4a3392ce87e3c0eab864a0b05b1cd7ec8a860a9803a975beba1dfR65